README.md

```diff
@@ -38,16 +38,14 @@ In addition to being an engaging education resource it has evolved into
 an *extremely powerful* and performance-ready *live coding instrument* suitable 
 for professional artists and DJs.
 
-Whilst Education is a core focus it now sits at the intersection
-between three core domains:
+While Education is the main priority, Sonic Pi aims to find a balance between
+three core domains:
 
 * **Art** - providing the means to express yourself and ask new questions of music and notation
-* **Technology** - exploring questions related to liveness, time and concurrency in programming languages
+* **Technology** - exploring questions related to liveness, time, and concurrency in programming languages
```

Comment:
I would actually argue that adding a serial comma here changes the interpretation of the sentence undesirably. Without it, the sentence reads more like '... questions related to [three aspects] in programming languages'. *With* the serial comma, to me it reads more like '... questions related to [two things], and [concurrency in programming languages]'.